docs/src/parallelization.md

```diff
@@ -167,8 +167,17 @@ which are both included in [`ode_default_options`](@ref).
 Trixi.jl allows parallel I/O using MPI by leveraging parallel HDF5.jl. To enable this, you first need
 to use a system-provided MPI library, see also [here](@ref parallel_system_MPI) and you need to tell
 [HDF5.jl](https://github.com/JuliaIO/HDF5.jl) to use this library.
-To do so, set the environment variable `JULIA_HDF5_PATH` to the local path
-that contains the `libhdf5.so` shared object file and build HDF5.jl by executing `using Pkg; Pkg.build("HDF5")`.
+To do so, set the preferences `libhdf5` and `libhdf5_hl` to the local paths of the libraries `libhdf5` and `libhdf5_hl`,
+which can be done by
+```julia
+julia> using Preferences, UUIDs
+julia> set_preferences!(
+           UUID("f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"), # UUID of HDF5.jl
+           "libhdf5" => "/path/to/your/libhdf5.so",
+           "libhdf5_hl" => "/path/to/your/libhdf5_hl.so", force = true)
+```
 For more information see also the [documentation of HDF5.jl](https://juliaio.github.io/HDF5.jl/stable/mpi/).
+In total, you should have a file called LocalPreferences.toml in the project directory that contains a section `[MPIPreferences]`,
+a section `[HDF5]` with entries `libhdf5` and `libhdf5_hl` as well as a section `[P4est]` with the entry `libp4est`.
 
 If you do not perform these steps to use parallel HDF5 or if the HDF5 is not MPI-enabled, Trixi.jl will fall back on a less efficient I/O mechanism. In that case, all disk I/O is performed only on rank zero and data is distributed to/gathered from the other ranks using regular MPI communication.
```

Comment:
I am not quite sure about this part anymore since the new HDF5_jll release also supports parallel HDF5 support. However, I am not sure if parallel HDF5 is always the default in HDF5_jll (on my system I have parallel support with HDF5_jll).

Reply:
Maybe you can update it? I think the JLL version is pinned to the HDF5 version, so you could say something along the lines of HDF5 > SOME_VERSION will provide parallel HDF5 on *most* (all?) systems, and that these steps might not be necessary anymore

Reply:
I definetely agree on updating it. I was only unsure to state that this is not necessary anymore for HDF5.jl >= v0.17 or to be a bit more reluctant and state this is not necessary anymore on *most* systems.

Reply:
Yeah, stating that it should not be necessary on most (default) systems sounds good to me :+1:

Reply:
Done in 40cede457c18bb4a8796e22c9771ab9946305029. I put that at the top of the section because there, we were previously already saying that HDF5_jll wouldn't provide parallel HDF5.